test/jdk/java/awt/Mixing/AWT_Mixing/OverlappingTestBase.java

```diff
@@ -382,10 +382,9 @@ protected void clickAndBlink(Robot robot, Point lLoc) {
     protected String failMessage = "The LW component did not received the click.";
 
     private static boolean isValidForPixelCheck(Component component) {
-        if ((component instanceof java.awt.Scrollbar) || isMac && (component instanceof java.awt.Button)) {
-            return false;
-        }
-        return true;
+        return !(component == null ||
+                (component instanceof java.awt.Scrollbar) ||
+                (isMac && (component instanceof java.awt.Button)));
```

Comment:
```suggestion
        return !(component instanceof java.awt.Scrollbar)
               && (!isMac || (!(component instanceof java.awt.Button)));
```
The expression can be simplified further.

`java.awt.Scrollbar` is already imported, so `Scrollbar` can safely be used. I'm for importing `java.awt.Button` and using unqualified `Button` in the expression.